src/store/queue/store.test.js

```diff
@@ -132,7 +133,14 @@ describe('watchPause', () => {
 })
 
 describe('watchNext', () => {
-  it('plays the next track', async () => {
+  it('loads random songs into the queue when last song is reached', async () => {
+    const randomTracks = [
```

Comment:
nit, technically we could just call this autoplayTracks since the behavior we're testing doesn't actually have to do with *how* we're getting the tracks, just that the getQueueAutoplay method is used

Reply:
Love it